app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CommentsController < ApplicationController
+  def index
+    @comments = Comment.all
+  end
+
+  def new
+    @comment = Comment.new
+  end
+
+  def comment_params
+    params.require(:comment).permit(:text)
+  end
```

Comment:
[OPTIONAL] Your strong parameters should be in a private method so that external parties wont have access to them